rites/roman1969/src/catalog/martyrology.ts

```diff
@@ -660,6 +660,15 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       dateOfDeath: 547,
       dateOfDeathIsApproximative: true,
     },
+    // src: https://www.cnbb.org.br/missal-romano-calendario-proprio-dos-santos-brasil/, https://en.wikipedia.org/wiki/Benedict_the_Moor
+    benedict_the_moor_religious: {
+      canonizationLevel: CanonizationLevels.Saint,
+      name: 'Benedict the Moor',
+      titles: [Title.Religious],
+      dateOfBirth: '1524-03-31',
+      dateOfDeath: '1589-04-04',
+      dateOfCanonization: '1807-05-24',
+    },
```

Comment:
You have missed two properties from the [previous PR](https://github.com/romcal/romcal/blob/24c3bd109906d49fae0781ca4f0bef076a3453a9/rites/roman1969/src/catalog/martyrology.ts#L1239-L1248) (`sex` and `dateOfBeatification`).

```suggestion
    benedict_the_moor_religious: {
      canonizationLevel: CanonizationLevels.Saint,
      dateOfBeatification: '2011-05-22',
      dateOfBirth: '1524-03-31',
      dateOfCanonization: '1807-05-24',
      dateOfDeath: '1589-04-04',
      name: 'Benedict the Moor',
      titles: [Title.Religious],
      sex: 'FEMALE',
    },
```